test/exception-handling.wast.fromBinary

```diff
@@ -15,48 +17,82 @@
   (nop)
  )
  (func $eh_test
-  (local $exn exnref)
+  (local $x i32)
+  (local $1 i64)
+  (local $2 (i32 i64))
+  (local $3 i32)
+  (local $4 i32)
   (try
    (do
     (throw $event$0
      (i32.const 0)
     )
    )
-   (catch
-    (local.set $exn
-     (pop exnref)
-    )
+   (catch $event$0
     (drop
-     (block $label$3 (result i32)
-      (rethrow
-       (br_on_exn $label$3 $event$0
-        (local.get $exn)
+     (pop i32)
+    )
+   )
+  )
+  (try
+   (do
+    (throw $event$2
+     (i32.const 0)
+     (i64.const 0)
+    )
+   )
+   (catch $event$2
```

Comment:
A simple `catch` with multivalue event in exception-handling.wast turns into this big code block after binary roundtripping. Is this expected? @tlively

Reply:
Yes, this is expected. Until I get Poppy IR landed, we will continue to emit terrible code for any multivalue construct and that code gets worse and worse each time it is parsed into Binaryen IR or emitted to a binary